src/components/QuestDetails/index.module.scss

```diff
@@ -60,3 +60,23 @@
   height: 100%;
   gap: var(--space-xl);
 }
+
+.endDate {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: var(--space-2xs);
+  color: var(--color-neutral-100);
+}
+
+.futureDate {
+  color: var(--color-primary-200);
```

Comment:
can you add a semantic css variable `--text-brand` here https://github.com/HyperPlay-Gaming/hyperplay-ui/blob/607e8abddac48bc89c460c62e17c4063f8da8cf8/src/styles/designSystem/_colors.scss#L106-L122

Also correct me if I'm wrong, but I believe it is `--color-primary-300` on figma https://www.figma.com/design/yHrBPoE70k1I2icW4paEit/Quests?node-id=1356-85942&t=Po4c1ophjvXFfYm5-4